repose-aggregator/src/docs/asciidoc/recipes/multi-tenant.adoc

```diff
@@ -0,0 +1,99 @@
+= Implementing Multi-tenant Authorization
+
+This recipe shows you how to configure your filters to support rest resources that contain multiple tenants in the uri
+all of which need authorization applied to them.
+
+For the remainder of this recipe we are going to assume that you are trying to secure a resource this resource: `/a_tenant/do/something/on/another_tenant`.
+
+== Derive Tenants
+
+The first thing to do is to use the <<../filters/url-extractor-to-header.adoc#,URL Extractor to Header filter>> to pull out both tenants we care about into a common header.
```

Comment:
If we are going to reference the `URL Extractor to Header filter` in this recipe, then it should probably be fleshed out too.